test/mergeWorldAndLocalPosition.js

```diff
@@ -0,0 +1,19 @@
+import 'babel-polyfill';
+import mergePositions from '../src/mergeWorldAndLocalPosition.es6.js';
+import assert from 'assert';
+
+describe("mergeWorldAndLocalPosition.js", () => {
+  it("is a function", () => {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["it" is not defined.](https://www.codacy.com/app/lagora-franck/proc-edu-ria/file/2543176237/issues/source?bid=3286583&fileBranchId=3287066#l6)